examples/pipelines/ccfraud/config.yaml

```diff
@@ -8,6 +8,8 @@ aml:
 
 # federated learning parameters
 federated_learning:
+  run_data_analysis: true
```

Comment:
could you move this parameter under either data_analysis_parameters or training_parameters ?

federated_learning is really a section that is only about configuring the silos and orchestrator.

Reply:
also, can we turn it off by default? (I think we need to document it better if we want to turn it on by default)